content/flux/v0.x/get-started/syntax-basics.md

```diff
@@ -0,0 +1,353 @@
+
+s // Returns foo
+i // Returns 1
+f // Returns 2
+```
+
+Variables can be assigned to any [Flux data type](/flux/v0.x/data-types/).
+
+## Data types
+Flux supports many data types grouped into the following categories:
+
+- [Basic types](#basic-types)
+- [Composite types](#composite-types)
+- [Regular expression types](#regular-expression-types)
+
+### Basic types
+The following basic types can be represented with literal values:
+
+- [Boolean](/flux/v0.x/data-types/basic/bool/)
+- [Duration](/flux/v0.x/data-types/basic/duration/)
+- [String](/flux/v0.x/data-types/basic/string/)
+- [Time](/flux/v0.x/data-types/basic/time/)
+- [Float](/flux/v0.x/data-types/basic/float/)
+- [Integer](/flux/v0.x/data-types/basic/integer/)
+
+```js
+// Boolean
+true
+
+// Duration
+23h4m5s
+
+// String
+"foo"
+
+// Time
+2021-01-01T00:00:00Z
+
+// Float
+1.0
+
+// Integer
+1
+```
+
+The following basic types do not have a literal syntax, but can be created in other ways.
+
+- [Bytes](/flux/v0.x/data-types/basic/bytes/)
+- [Unsigned integers](/flux/v0.x/data-types/basic/uint/)
+- [Nulls](/flux/v0.x/data-types/basic/null/)
+
+### Composite Types
+Flux [composite types](/flux/v0.x/data-types/composite/) are constructed from
+Flux [basic types](#basic-types).
+
+- [Records](#records)
+- [Arrays](#arrays)
+- [Dictionaries](#dictionaries)
+- [Functions](#functions)
+
+#### Records
+A **record** is a collections of key-value pairs.
+Each key is a string.
+Each value can be a different data type.
+
+```js
+{name:"Jim", age: 42, "favorite color": "red"}
+```
+
+Use **dot notation** or **bracket notation** to access a properties of a record:
+
+{{% note %}}
+Use bracket notation to reference record properties with special or
+white space characters in the property key.
+{{% /note %}}
+
+```js
+o = {name:"Jim", age: 42, "favorite color": "red"}
+
+o.name
+// Returns Jim
+
+o.age
+// Returns 42
+
+o["favorite color"]
+// Returns red
+```
+
+_For more information, see [Work with records](/flux/v0.x/data-types/composite/record/)._
+
+#### Arrays
+An **arrays** is a collection of values of the same type.
+
+```js
+n = 4
+l = [1,2,3,n]
+
+l
+// Returns [1, 2, 3, 4]
+```
+
+Use **bracket notation** to access a value at a specific index in an array:
+
+```js
+a = ["foo","bar","baz","quz"]
+
+a[0]
+// Returns foo
+```
+
+_For more information, see [Work with arrays](/flux/v0.x/data-types/composite/array/)._
+
+#### Dictionaries
+A **dictionary** is a collection of key-value pairs with keys of the same type
+and values of the same type.
+
+```js
+[1: "foo", 2: "bar"]
+```
+
+Use [`dict.get()`](/flux/v0.x/stdlib/dict/get/) to access elements in a dictionary:
+
+```js
+import "dict"
+
+d = [1: "foo", 2: "bar"]
+
+dict.get(dict: d, key: "1", default: "")
+// Returns foo
+```
+
+_For more information, see [Work with dictionaries](/flux/v0.x/data-types/composite/dict/)._
+
+#### Functions
+A **function** is a set of parameters that perform an operation.
```

Comment:
Conceptual nit: the parameters don't perform the operation, the function does!
Maybe "A function is a rule for taking input parameters to an output."